trk/src/App.js

```diff
@@ -1,30 +1,29 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import { Provider as PaperProvider } from 'react-native-paper';
 import auth from '@react-native-firebase/auth';
 import AppNavigator from './Navigation/AppNavigator';
 import AuthNavigator from './Navigation/AuthNavigator';
+import { AuthProvider, AuthContext } from './Utils/AuthContext';
 
-function App(props) {
-  const [initializing, setInitializing] = useState(true);
-  const [user, setUser] = useState();
 
-  function onAuthStateChanged(user) {
-    setUser(user);
-    if (initializing) setInitializing(false);
-  }
+function AppWrapper() {
```

Comment:
To make the User 'context' available throughout the code (i.e. access user data evrywhr), I had to include an Wrapper around the app. For some reason, wrapping the <App /> component in <AuthProvider /> alone wasn't enough (down below). Idk why, but I needed to create a wrapper around the App function.